### PR TITLE
feat(EU Covid Certificate): [IAGP-35] Certificate in wrong format

### PR DESCRIPTION
**Pull request by debiff**
## Short description
This PR add the screen that inform the user that the selected certificate is in the wrong format (response code 400)

## List of changes proposed in this pull request
- Added the certificate not found screen (`EuCovidCertWrongFormatKoScreen.tsx`)

## How to test
Run `EuCovidCertNotFoundKoScreen.test.tsx ` with Jest.
On dev server `src/routers/features/eu_covid_cert/index.ts` return 400

### CODE REVIEW

File: locales/en/index.yml [hunk 1]
```diff
@@ -2457,17 +2459,19 @@ features:
...
       notOperational:
         title: "The european COVID certificate is not currently operational"
         subtitle: "TMP"
       temporarilyNotAvailable:
         title: "The service is temporarily unavailable"
         subtitle: "Please try again later"
       wrongFormat:
-        title: "TMP"
-        subtitle: "TMP"
+        title: "This certificate cannot be \nrecovered: contact support"
```
Comment (thisisjp) [inline locales/en/index.yml:2472]:
```suggestion
        title: "This certificate cannot be \nretrieved: contact support"
```

File: locales/en/index.yml [hunk 2]
```diff
@@ -2457,17 +2459,19 @@ features:
         title: "TMP"
         subtitle: "TMP"
       notFound:
-        title: "TMP"
-        subtitle: "TMP"
+        title: "The certificate does not appear to \nexist: contact support"
```
Comment (thisisjp) [inline locales/en/index.yml:2462]:
```suggestion
        title: "It seems that the certificate does not \nexist: contact support"
```

**Review by thisisjp — CHANGES_REQUESTED** [review]
Not sure about the correct position of the line break, as I’ve tweaked a little bit the wording

File: locales/en/index.yml (hunk 2 above)
Comment (debiff) [reply locales/en/index.yml:2469]: Here the result:

File: locales/en/index.yml (hunk 1 above)
Comment (debiff) [reply locales/en/index.yml:2479]: Here the result:

File: ts/features/euCovidCert/screens/ko/EuCovidCertWrongFormatKoScreen.tsx [hunk 3]
```diff
@@ -1,22 +1,124 @@
...
-import { H1 } from "../../../../components/core/typography/H1";
+import { Image, StyleSheet } from "react-native";
+import { View } from "native-base";
 import { GlobalState } from "../../../../store/reducers/types";
 import { BaseEuCovidCertificateLayout } from "../BaseEuCovidCertificateLayout";
+import { H4 } from "../../../../components/core/typography/H4";
+import CopyButtonComponent from "../../../../components/CopyButtonComponent";
+import { EUCovidCertificateAuthCode } from "../../types/EUCovidCertificate";
+import { InfoScreenComponent } from "../../../../components/infoScreen/InfoScreenComponent";
+import wrongFormatImage from "../../../../../img/messages/error-message-detail-icon.png";
```
Comment (fabriziofff) [inline ts/features/euCovidCert/screens/ko/EuCovidCertWrongFormatKoScreen.tsx:12]: Could you add also the red version of this image?
According to the design, the image must be red

**Review by fabriziofff — CHANGES_REQUESTED** [review]

File: ts/features/euCovidCert/screens/ko/EuCovidCertWrongFormatKoScreen.tsx (hunk 3 above)
Comment (debiff) [reply ts/features/euCovidCert/screens/ko/EuCovidCertWrongFormatKoScreen.tsx:12]: No problem for me, I used the other version in agreement with @thisisjp.

Comment (debiff) [reply ts/features/euCovidCert/screens/ko/EuCovidCertWrongFormatKoScreen.tsx]: Modified in [78e5b84](https://github.com/pagopa/io-app/pull/3093/commits/78e5b84850ee7b87a300e8e9e8938f41f6e6f918)

**Review by fabriziofff — APPROVED** [review]

**Comment by fabriziofff** [comment]
lgfm, waiting for @thisisjp approval

**Review by thisisjp — APPROVED** [review]

**Pull request merged**